### PR TITLE
DL - Initial include prod

### PR DESCRIPTION
**Pull request by douglasrlee**
Based off of a combination of:
* https://github.com/strongloop/strong-mocha-interfaces
* https://github.com/mochajs/mocha/blob/master/lib/interfaces/bdd.js
* https://gist.github.com/jgonera/1477482

This will need to be built as a package and pushed to GitHub packages or just regular npm, as a public package, and then when you install it into your project you will run it like so: `mocha --require <package name> --ui env-bdd`.

Tests when `ENVIRONMENT` isn't `prod` or `production`:
Screen Shot 2020-12-01 at 3 44 12 PM

Tests when `ENVIRONMENT` is `prod` or `production`:
Screen Shot 2020-12-01 at 3 43 07 PM

### CODE REVIEW

File: .node-version [hunk 1]
```diff
@@ -0,0 +1 @@
+10.16.2
```
Comment (dannyburrows) [inline .node-version:1]: At least take this to 12.x

**Review by dannyburrows — APPROVED** [review]
V. nice

File: .node-version (hunk 1 above)
Comment (douglasrlee) [reply .node-version:1]: oh yeah.. didn't realize this was that old

**Review by sgelbart — APPROVED** [review]

**Comment by hobe-dev** [comment]
very cool

**Review by hobe-dev — APPROVED** [review]

**Pull request merged**